spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootContextLoader.java

```diff
@@ -141,7 +141,7 @@ private void assertHasClassesOrLocations(MergedContextConfiguration mergedConfig
 		boolean hasClasses = !ObjectUtils.isEmpty(mergedConfig.getClasses());
 		boolean hasLocations = !ObjectUtils.isEmpty(mergedConfig.getLocations());
 		Assert.state(hasClasses || hasLocations,
-				() -> "No configuration classes or locations found in @SpringApplicationConfiguration. "
+				() -> "No configuration classes or locations are found in @SpringApplicationConfiguration. "
```

Comment:
I don't think "are" is needed here.

Reply:
Changes have been made, thank you for the feedback as always.